UITests/ClearPrivateDataTests.swift

```diff
@@ -100,18 +100,19 @@ class ClearPrivateDataTests: KIFTestCase, UITextFieldDelegate {
         
         // Only one will be found -- we collapse by domain.
         anyDomainsExistOnTopSites(dispDomains, fulldomains: fullDomains)
-
         BrowserUtils.clearPrivateData([BrowserUtils.Clearable.History], swipe: false, tester: tester())
-        EarlGrey.select(elementWithMatcher: grey_accessibilityLabel(urls[0].title))
+    
+        // Tap bookmark and come back to refresh DOM
+        EarlGrey.select(elementWithMatcher:  grey_accessibilityID("HomePanels.Bookmarks")).perform(grey_tap())
+        EarlGrey.select(elementWithMatcher:  grey_accessibilityID("HomePanels.TopSites")).perform(grey_tap())
+        
+        let displayed = urls[0].dispDomain.replacingOccurrences(of: ".", with: " ")
+        EarlGrey.select(elementWithMatcher: grey_accessibilityLabel(displayed))
```

Comment:
Can you comment what this code is doing? I think it is trying to tap a cell in the top sites collection view by name. Curious how this works, if the url is 'facebook.com' the `displayed` becomes 'facebook com' (no dot), but doesn't the TLD part (.com) also need to be removed?